cypress/support/step_definition/login_management/commonLoginManagementRelatedSteps.spec.js

```diff
@@ -0,0 +1,11 @@
+import Login_Management from '../../pages/login_management/pageLoginManagement'
+
+const loginManagement = new Login_Management()
+
+Given('I am on the dBildungscloud login page as a user on BRB', () => {
```

Comment:
Make it a generic step and not specific to only BRB

Reply:
updated!